static/translations/messages_en.json

```diff
@@ -5,6 +5,14 @@
   "Private messages": "Private messages",
   "Starred": "Starred",
   "Mentions": "Mentions",
+  "Mentions ({count} unread)":"Mentions ({count} unread)",
+  "Stream: {streamName}":"Stream: {streamName}",
+  "Stream: {streamName} ({unreadCount} unread messages)":"Stream: {streamName} ({unreadCount} unread messages)",
+  "Topic: {name}":"Topic: {name}",
+  "Topic: {name} ({unreadCount} unread messages)":"Topic: {name} ({unreadCount} unread messages)",
+  "Back":"Back",
+  "Navigate up":"Navigate up",
+  "Starred messages":"Starred messages",
```

Comment:
Nit (for all these added entries): put a space after the colon, like in the rest of the file. So, `"...": "..."` instead of `"...":"..."`.